src/app/components/experiment-workspace-detail/experiment-workspace-detail.component.ts

```diff
@@ -6,11 +6,13 @@ import { filter } from 'rxjs/operators';
 import { DeleteDialog } from 'src/app/dialogs/delete-dialog/delete-dialog.dialog';
 import { ApiLink, CollectionApiObject, PageApiObject } from 'src/app/services/api-data-types';
 import { PluginRegistryBaseService } from 'src/app/services/registry.service';
-import { TemplateTabApiObject } from 'src/app/services/templates.service';
+import { TemplateTabApiObject, TemplatesService } from 'src/app/services/templates.service';
 import { TemplateApiObject } from 'src/app/services/templates.service';
 import { COMMA, ENTER } from '@angular/cdk/keycodes';
 import { MatChipInputEvent } from '@angular/material/chips';
 import { Subscription } from 'rxjs';
+import { TAB_GROUP_NAME_OVERRIDES, TAB_GROUP_SORT_KEYS } from '../tab-group-list/tab-group-list.component';
```

Comment:
It is better to push these constants to a sensible third location if they are used multiple times (e.g., into the templates service?)